### PR TITLE
Create verify-signed-images.md

### PR DESCRIPTION
**Pull request by ihuibin**
Signed-off-by: bin.hui <bin.hui@daocloud.io>

### CODE REVIEW

**Review by Sea-n — CHANGES_REQUESTED** [review]
Tips: You can use machine translator to confirm what the original content means.

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+```shell
+COSIGN_EXPERIMENTAL=1 cosign verify k8s.gcr.io/kube-apiserver-amd64:v1.24.0
+```
+
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` is used to allow verification of images signed
+in `KEYLESS` mode. To learn more about keyless signing, please refer to
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)
+. {{< /note >}}
```
[the remark this answers is not in the document]
Comment (ihuibin) [reply content/zh/docs/tasks/administer-cluster/verify-signed-images.md]: OK，I translated it.

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+These instructions are for Kubernetes {{< skew currentVersion >}}. If you want
+to check the integrity of components for a different version of Kubernetes,
+check the documentation for that Kubernetes release.
+
+You will need to have the following tools installed:
+
+- `cosign` ([install guide](https://docs.sigstore.dev/cosign/installation/))
+- `curl` (often provided by your operating system)
+-->
+这些说明适用于 {{< skew currentVersion >}}。如果你想要检查不同版本完整的 Kubernetes 组件，请查看该 Kubernetes 版本的文档。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:28]:
```suggestion
这些说明适用于 {{< skew currentVersion >}}。如果你想要检查其他版本的 Kubernetes 组件的完整性，
请查看对应 Kubernetes 版本的文档。
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+你需要安装以下工具:
+- `cosign` ([安装指导](https://docs.sigstore.dev/cosign/installation/))
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:30]:
```suggestion
- `cosign` ([安装指南](https://docs.sigstore.dev/cosign/installation/))
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` is used to allow verification of images signed
+in `KEYLESS` mode. To learn more about keyless signing, please refer to
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)
+. {{< /note >}}
+-->
+
+## 验证镜像签名 {#verifying-image-signatures}
+
+完整的镜像签名列表请参见链接 [Releases](/releases/download/).
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:55]:
```suggestion
完整的镜像签名列表请参见[发行版本](/releases/download/)。
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+
+我们从这个列表中选择一个镜像，并使用 `cosign verify` 命令来验证它的签名：
+
+
+```shell
+COSIGN_EXPERIMENTAL=1 cosign verify k8s.gcr.io/kube-apiserver-amd64:v1.24.0
+```
+
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` 用于在 `KEYLESS` 模式对已签名的镜像进行验证。 想获取更多`KEYLESS`， 请参考
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:65]:
```suggestion
`COSIGN_EXPERIMENTAL=1` 用于对以 `KEYLESS` 模式签名的镜像进行验证。想要进一步了解 `KEYLESS`，请参考
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)
+. {{< /note >}}
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:67]:
```suggestion
[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)。
{{< /note >}}
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+example: `registry-url/image-name@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`
+.
+
+For more information, please refer
+to [Image Pull Policy](/docs/concepts/containers/images/#image-pull-policy)
+section.
+-->
+### 验证所有控制平面组件镜像
+
+验证所有控制平面组件镜像，请运行以下命令：
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:93]:
```suggestion
验证所有已签名的控制平面组件镜像，请运行以下命令：
```

Comment (tengqm) [reply content/zh/docs/tasks/administer-cluster/verify-signed-images.md:93]: 上面几行可以单独翻译，接下来的代码段直接复用英文的就好，不必注释一份再复制一份。

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+```shell
+curl -Ls https://sbom.k8s.io/$(curl -Ls https://dl.k8s.io/release/latest.txt)/release | grep 'PackageName: k8s.gcr.io/' | awk '{print $2}' > images.txt
+input=images.txt
+while IFS= read -r image
+do
+  COSIGN_EXPERIMENTAL=1 cosign verify "$image"
+done < "$input"
+```
+
+当你验证一个镜像，在你的 Pod 清单通过指定该镜像的杂凑值，像是此范例：`registry-url/image-name@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:104]:
```suggestion
当你完成某个镜像的验证时，可以在你的 Pod 清单通过摘要值来指定该镜像，例如：
`registry-url/image-name@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`。
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+.
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:105]: 删除此行

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 10]
```diff
@@ -0,0 +1,130 @@
...
+e.g. [conformance image](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/image/README.md))
+, signatures can also be verified at deploy time using
+[cosigned](https://docs.sigstore.dev/cosign/kubernetes/#cosigned-admission-controller)
+admission controller. To get started with `cosigned` here are a few helpful
+resources:
+
+* [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/cosigned)
+* [Configuration Options](https://github.com/sigstore/cosign/tree/main/config)
+-->
+## 使用准入控制器验证镜像
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:121]:
```suggestion
## 使用准入控制器验证镜像签名
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 11]
```diff
@@ -0,0 +1,130 @@
...
+
+有一些非控制平面镜像（例如
+[conformance image](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/image/README.md)
+），在部署时签名也可以使用
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:125]:
```suggestion
），也可以在部署时使用
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 12]
```diff
@@ -0,0 +1,130 @@
...
+[cosigned](https://docs.sigstore.dev/cosign/kubernetes/#cosigned-admission-controller)
+控制器进行验证。如要使用 “cosigned”，下面是一些有帮助的资源：
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:127]:
```suggestion
控制器验证其签名。如要使用 “cosigned”，下面是一些有用的资源：
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+These instructions are for Kubernetes {{< skew currentVersion >}}. If you want
+to check the integrity of components for a different version of Kubernetes,
+check the documentation for that Kubernetes release.
+
+You will need to have the following tools installed:
+
+- `cosign` ([install guide](https://docs.sigstore.dev/cosign/installation/))
+- `curl` (often provided by your operating system)
+-->
+这些说明适用于 {{< skew currentVersion >}}。如果你想要检查其他版本的 Kubernetes 组件的完整性，
```
Comment (Sea-n) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:28]:
```suggestion
这些说明适用于 Kubernetes {{< skew currentVersion >}}。如果你想要检查其他版本的 Kubernetes 组件的完整性，
```

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+* [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/cosigned)
+* [Configuration Options](https://github.com/sigstore/cosign/tree/main/config)
+-->
+## 使用准入控制器验证镜像签名
+
+有一些非控制平面镜像（例如
+[conformance image](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/image/README.md)
+），也可以在部署时使用
+[cosigned](https://docs.sigstore.dev/cosign/kubernetes/#cosigned-admission-controller)
+控制器验证其签名。如要使用 “cosigned”，下面是一些有用的资源：
```
Comment (Sea-n) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:129]:
```suggestion
控制器验证其签名。如要使用 `cosigned`，下面是一些有用的资源：
```

**Comment by ihuibin** [comment]
/assign 
Signed-off-by: bin.hui <bin.hui@daocloud.io>

**Comment by sftim** [comment]
According to Prow, this PR has an invalid commit message.
/approve cancel

Once that's fixed, re-approval is fine.

**Comment by tengqm** [comment]
Why we have so many commits in this PR? It could have been a very clean PR because it only touches a single, non-existent file.

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md (hunk 9 above)
Comment (ihuibin) [reply content/zh/docs/tasks/administer-cluster/verify-signed-images.md]: 好的

**Comment by tengqm** [comment]
/close
Please propose a new PR for this. This one is currently messy.

**Pull request closed without merging**